src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -229,7 +230,14 @@ private static IPAddress[] DetermineServerIp(Uri serverUri)
                 if (getHostAddressesTask.Wait(TimeSpan.FromSeconds(30)) == false)
                     throw new InvalidOperationException($"Could not obtain IP address from DNS {serverUri.DnsSafeHost} for 30 seconds.");
```

Comment:
This will also kill us, no?

Reply:
Yes. We're throwing in multiple places during `SecurityConfiguration.Validate`. IMO this is not bad, we're preventing a server from running with invalid security settings.